ADOPTERS.md

```diff
@@ -9,6 +9,7 @@ KubeEdge is used by many companies. If you are using KubeEdge in your organizati
 <a href="http://xh-iot.net/" border="0" target="_blank"><img alt="XH-iot" src="docs/adopters/xh-iot.png" height="70"></a> &nbsp; &nbsp; &nbsp;
 <a href="http://www.jylink.com/" border="0" target="_blank"><img alt="jylink" src="docs/adopters/jylink.png" height="70"></a> &nbsp; &nbsp; &nbsp;<br/><br/>
 <a href="http://www.ictnj.ac.cn/" border="0" target="_blank"><img alt="ictnj" src="docs/adopters/ictnj.svg " height="70"></a>&nbsp; &nbsp; &nbsp;<br/><br/>
+<a href="https://www.daocloud.io/" border="0" target="_blank"><img alt="daocloud" src="docs/adopters/daocloud.svg " height="70"></a>&nbsp; &nbsp; &nbsp;<br/><br/>
```

Comment:
Need to remove extra line breaks.
![image](https://user-images.githubusercontent.com/49225405/224262041-2b73d307-aeb1-45e7-b005-d992de3e3743.png)

Reply:
done